packages/components/Shimmer/ios/MSFShimmerViewManager.m

```diff
@@ -0,0 +1,9 @@
+#import <React/RCTViewManager.h>
+#import <React/RCTConvert.h>
+
+@interface RCT_EXTERN_MODULE(MSFShimmerViewManager, RCTViewManager)
+
+RCT_EXPORT_VIEW_PROPERTY(appearance, NSDictionary)
```

Comment:
Can you comment what this is and how it's used? Same for shimmerAppearance

Reply:
So, duplicate the documentation from FUA / from Shimmer.tsx ? Since the same property is documented on both ends

Reply:
I'm just gonna add comments everywhere. Prefer overcomment to undercomment I guess